spec/features/edit_course_details_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+feature "edit course details" do
+  scenario "edit with valid parameters" do
+    given_a_trainee_exists
+    when_i_visit_the_course_details_page
+    and_i_enter_valid_parameters
+    then_i_am_redirected_to_the_summary_page
+  end
+
+  def given_a_trainee_exists
+    @trainee = create(:trainee)
+  end
+
+  def when_i_visit_the_course_details_page
+    visit "/trainees/#{@trainee.id}/course-details"
```

Comment:
Are we able to load the page using the page object? ie: https://github.com/DFE-Digital/register-trainee-teacher-data/blob/e1e292500a2681665d9310e6a1133d8f73165366/spec/features/trainees/trainee_summary_spec.rb#L20

Reply:
We had some problems with the `load` method on Find. I think it was because Addressable didn't support rails format parameters (array querystrings etc) or something like that so we ditched it. I also think it's more readable as you don't need to go to the page object to see what URL you're testing.

Reply:
🤔 could it be that Find was doing some weird csharp to rails conversion etc somewhere which would have caused the problem?

Reply:
could we retry and if its really and issue then switch it just for those instances

Reply:
It wasn't a C# thing it was the rails format it didn't like. There's no way to set array parameters in the hash you pass to load. We're not using array parameters here anyway so I'll change it. I still prefer not using it as it's more explicit to have the URL in the spec but 🤷‍♂️

Reply:
Updated in f09c23d